File: testautomation/testautomation/Test/AdminPage.cs
using System;
using System.Text;
using System.Text.RegularExpressions;
using System.Threading;
using NUnit.Framework;
using OpenQA.Selenium;
using OpenQA.Selenium.Firefox;
using OpenQA.Selenium.Chrome;
using OpenQA.Selenium.Support.UI;

namespace PageObjectPattern
{
    internal class AdminPage :Page
    {
        public AdminPage(IWebDriver driver) : base(driver)
        {
        }

        internal PostPage goToAddNewPostPage()
        {
            driver.FindElement(By.Id("wp-admin-bar-ab-new-post")).Click();
           
            return new PostPage(driver);
        }
    }
}

File: src/main/java/pages/AdminPage.java
package pages;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;


public class AdminPage extends Page {


    public AdminPage(WebDriver driver) {
        super(driver);
    }

    public void open() {
        driver.get(baseUrl + "wp-admin");
    }
}
